diff --git a/examples/CMakeLists.txt b/examples/CMakeLists.txt
index c5ce5fd..200acfa 100644
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -46,11 +46,25 @@ build_lib_example(
 )
 
 # Also requires psc and sip modules; uncomment the below if they are present
-#build_lib_example(
-#  NAME nr-prose-l3-relay-mcptt
-#  SOURCE_FILES nr-prose-l3-relay-mcptt.cc
-#  LIBRARIES_TO_LINK ${PROSE_L3_RELAY_ON_OFF_LIBRARIES}
-#                    ${libnr-prose}
-#                    ${libpsc}
-#                    ${libsip}
-#)
+build_lib_example(
+  NAME nr-prose-l3-relay-mcptt
+  SOURCE_FILES nr-prose-l3-relay-mcptt.cc
+  LIBRARIES_TO_LINK ${PROSE_L3_RELAY_ON_OFF_LIBRARIES}
+                    ${libnr-prose}
+                    ${libpsc}
+                    ${libsip}
+)
+
+
+build_lib_example(
+  NAME nr-mcptt-operational-modes-static
+  SOURCE_FILES nr-prose-examples/nr-mcptt-operational-modes-static.cc
+               ${nr-prose-examples_source_files}
+  LIBRARIES_TO_LINK ${PROSE_L3_RELAY_ON_OFF_LIBRARIES}
+                    ${libnr}
+                    ${libnr-prose}
+                    ${libpsc}
+                    ${libsip}
+)
+
+#SHOULD I ADD the libnr line here for libraries to link
